### DISCUSSION TITLE
Notes in Tutanota

### DISCUSSION THREAD

**Discussion by TBDwarf** [tutao/tutanota#5245] [Ideas]
It would be really good to be able to take notes directly in Tutanota, as a model I think of services like ''standard notes''.
This little extra would be original because not present in other mail services and would perfectly complement the combo email, contact, calendar!
A note-taking rethinks little consumption of data.

**Comment by serrq** [comment]
I believe notes will follow when a "tutadrive" will be developed.

**Comment by vitoreiji** [comment]
That is a nice idea, we will consider it!

**Comment by serrq** [comment]
There would also be the voice call to close the communication suite.
I mean among email addresses.

**Reply by vitoreiji** [reply]
Also interesting, but a lot more challenging =D

We will keep this in mind, though, thanks!

**Reply by dyeray** [reply]
Not that I support this request, but if you consider it at some point, please consider just offering a Matrix server. No need to reinvent the wheel. It would be sad to have yet another half-baked open source end-to-end encrypted network to fragment the space.

+1 for the notes though :). In fact I think it would probably be possible to add a note taking UI on top of Matrix...

**Comment by chriscacao** [comment]
It would be great if you could share notes with selected Tutanota users in the future.

**Comment by quorend** [comment]
I currently use draft emails for taking notes. My TODO email for example I'm always adding items to and then removing them when I've completed them. I once experienced a bug in Tuta mail where I could open my TODO draft email and modify it, but when I closed the email as a draft again, the email draft could not be displayed without editing the draft again. Instead an error message was displayed. This occurred after many months of using this draft email. I ended up just copying the contents of the draft, creating a new email, copying into it, and deleting the old draft email. I have yet to experience the bug again, so I won't file an issue unless I see it again.

**Comment by tech2mar** [comment]
This is one of the most have!

**Comment by videbar** [comment]
Honestly, I don't want them to pursue something like this. I like having an email provider that is just an email provider. I understand that contacts are tightly integrated with email, and, because of historical and technical reasons, the calendar is bundled together as well, but I would prefer that they stay focus on offering the best service possible (e.g., native apps on all platforms) rather than trying to become an everything-service like google or proton.
